hipscat/util.py

```diff
@@ -6,6 +6,10 @@
 import os
 from bs4 import BeautifulSoup, SoupStrainer
 
+try:
+    from . import lsd2_io
+except ImportError:
+    import lsd2_io
```

Comment:
Why is this being imported like this?

Reply:
This is when I want to test functions by calling directly `python util.py` and it runs the function in the `if __name__ == '__main__':...` . Easy for quick prototyping debug. Will not persist to production.